### PR TITLE
Add contact info to Form Upload review page

### PR DESCRIPTION
**Pull request by Thrillberg**
## Summary
This PR adds the newly-added Contact Info to the review page on the Form Upload flow.
Screenshot 2024-12-11 at 11 00 43 AM

## Related issue(s)
https://github.com/orgs/department-of-veterans-affairs/projects/1443/views/3?sliceBy%5Bvalue%5D=Thrillberg&pane=issue&itemId=88758483&issue=department-of-veterans-affairs%7CVA.gov-team-forms%7C1895

### CODE REVIEW

File: src/applications/simple-forms/form-upload/components/CustomReviewTopContent.jsx [hunk 1]
```diff
@@ -53,6 +60,25 @@ const CustomReviewTopContent = () => {
     </div>
   );
 
+  const formattedPhoneNumber = () =>
+    `(${phoneNumber.substr(0, 3)}) ${phoneNumber.substr(
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/form-upload/components/CustomReviewTopContent.jsx:64]: I think phoneNumber can have dashes in it. Is that right? if so we can replace those out.

**Review by rhasselle-oddball — COMMENTED** [review]
looks good, one comment to check

File: src/applications/simple-forms/form-upload/components/CustomReviewTopContent.jsx (hunk 1 above)
Comment (Thrillberg) [reply src/applications/simple-forms/form-upload/components/CustomReviewTopContent.jsx]: Good point! Fixed.

**Review by rhasselle-oddball — APPROVED** [review]
looks good!

**Pull request merged**